### DISCUSSION TITLE
Flow chart

### DISCUSSION THREAD

**Discussion by castroflavio** [scistream/scistream-proto#37] [Ideas]
Is there a flow chart of the Scistream negotiation process?

Are there alternatives to Globus auth? external dependency might be a deal breaker, we haven't seen that client yet but it might exist.

**Comment by JoaquinChung** [comment]
![scistream-protocol-simple](https://github.com/user-attachments/assets/5fa156d7-fb36-490d-9dd4-b7dab0e5ba8f)
This is an old sequence diagram of the control protocol. We should update this with the current implementation.

![scistream-negotiation](https://github.com/user-attachments/assets/cb5ba405-6622-455f-ac70-3d028636cb4f)
This is the negotiation protocols that appears in the HPDC'22 paper.
